### PR TITLE
fix json error

### PR DESCRIPTION
**Pull request by jngz-es**
### Description
[Describe what this change achieves]
 
### Issues Resolved
[List any issues this PR will resolve]
 
### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
- [ ] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by dhrubo-os** [comment]
```
{"error":{"root_cause":[{"type":"mapper_parsing_exception","reason":"Root mapping definition has unsupported parameters:  [guardrails : {type=flat_object}]"}],"type":"mapper_parsing_exception","reason":"Failed to parse mapping [_doc]: Root mapping definition has unsupported parameters:  [guardrails : {type=flat_object}]","caused_by":{"type":"mapper_parsing_exception","reason":"Root mapping definition has unsupported parameters:  [guardrails : {type=flat_object}]"}},"status":400}
```

As we discussed we need to put this field inside of `properties`

File: common/src/main/java/org/opensearch/ml/common/CommonValue.java [hunk 1]
```diff
@@ -262,13 +262,13 @@ public class CommonValue {
                         + MLModel.LAST_UNDEPLOYED_TIME_FIELD
                         + "\": {\"type\": \"date\", \"format\": \"strict_date_time||epoch_millis\"},\n"
                         + "      \""
+                        + MLModel.GUARDRAILS_FIELD
```
Comment (ylwu-amzn) [inline common/src/main/java/org/opensearch/ml/common/CommonValue.java:265]: Have you verified if the index mapping is valid ? You can manually try it in OSD dev tools

**Comment by joshpalis** [comment]
> ```
> {"error":{"root_cause":[{"type":"mapper_parsing_exception","reason":"Root mapping definition has unsupported parameters:  [guardrails : {type=flat_object}]"}],"type":"mapper_parsing_exception","reason":"Failed to parse mapping [_doc]: Root mapping definition has unsupported parameters:  [guardrails : {type=flat_object}]","caused_by":{"type":"mapper_parsing_exception","reason":"Root mapping definition has unsupported parameters:  [guardrails : {type=flat_object}]"}},"status":400}
> ```
> 
> As we discussed we need to put this field inside of `properties`

This seems to be related to https://github.com/opensearch-project/OpenSearch/issues/12775

**Comment by dhrubo-os** [comment]
```
Tests with failures:
1106 tests completed, 1 failed, 58 skipped
 - org.opensearch.ml.task.MLPredictTaskRunnerTests.testExecuteTask_OnLocalNode_RemoteModel
 ```
 
 I think the core problem is solved. This is could be due to @Zhangxunmt's integration test failure?

**Review by dhrubo-os — APPROVED** [review]

**Review by Zhangxunmt — APPROVED** [review]

**Comment by Zhangxunmt** [comment]
Let's merge sooner than later to unblock other tests.

**Pull request merged**